configure

```diff
@@ -555,6 +562,7 @@ HAVE_CDDB=n
 CONFIG_TREMOR=n
 CONFIG_MIKMOD=n
 CONFIG_BASS=n
+CONFIG_OPENSSL=y
```

Comment:
Is there a reason why this isn't being left to automatic?

Reply:
No actual reason. It probably makes more sense to have it set to automatic, indeed.